src/client/sandbox/node/element.js

```diff
@@ -224,10 +233,12 @@ export default class ElementSandbox extends SandboxBase {
         if (ElementSandbox._isHrefAttrForBaseElement(el, attr))
             urlResolver.updateBase(getDestLocation(), this.document);
 
-        if (attr !== 'autocomplete')
-            return removeAttrFunc.apply(el, args);
+        var result = attr !== 'autocomplete' ? removeAttrFunc.apply(el, args) : void 0;
+
+        if (attr === 'onsubmit' && el.onsubmit === null)
+            this.setOnsubmit(el, null);
```

Comment:
Change to 
`if (attr === 'onsubmit' && typeof el.onsubmit !== 'function')`
I see client code where attribute handlers was removed as `el.onclick={}`.